packages/grafana-prometheus/src/datasource.ts

```diff
@@ -620,6 +625,11 @@ export class PrometheusDatasource
 
   // By implementing getTagKeys and getTagValues we add ad-hoc filters functionality
   async getTagValues(options: DataSourceGetTagValuesOptions<PromQuery>) {
+    return Promise.resolve([
+      { value: 'value', text: 'Value' },
+      { value: 'anotherValue', text: 'anotherValue' },
+    ]);
```

Comment:
Please remove this before merging ;)

Btw - for testing you can use static dimensions, variables can be configured with those directly in the UI.